js/jquery-accessibleMegaMenu.js

```diff
@@ -411,7 +438,7 @@ limitations under the License.
             target
                 .removeClass(this.settings.focusClass);
 
-            if (window.cvox) {
+            if (typeof window.cvox !== 'undefined' && window.cvox == true) {
```

Comment:
Feels a bit odd to test truthiness of ChromeVox object using `window.cvox == true`.